static/rison.js

```diff
@@ -1,516 +0,0 @@
-/* eslint-disable */
-// Taken from https://github.com/Nanonid/rison at e64af6c096fd30950ec32cfd48526ca6ee21649d (Jun 9, 2017)
-// Uses CommonJS, AMD or browser globals to create a module.
-// Based on: https://github.com/umdjs/umd/blob/master/commonjsStrict.js
-(function (root, factory) {
-    if (typeof define === 'function' && define.amd) {
-        // AMD. Register as an anonymous module.
-        define(['exports'], factory);
-    } else if (typeof exports === 'object') {
-        // CommonJS
-        factory(exports);
-    } else {
-        // Browser globals
-        factory((root.rison = {}));
-    }
-})(this, function (exports) {
-    var rison = exports;
-
-    //////////////////////////////////////////////////
-    //
-    //  the stringifier is based on
-    //    http://json.org/json.js as of 2006-04-28 from json.org
-    //  the parser is based on
-    //    http://osteele.com/sources/openlaszlo/json
-    //
-
-    if (typeof rison == 'undefined') window.rison = {};
-
-    /**
-     *  rules for an uri encoder that is more tolerant than encodeURIComponent
-     *
-     *  encodeURIComponent passes  ~!*()-_.'
-     *
-     *  we also allow              ,:@$/
-     *
-     */
-    rison.uri_ok = {
-        // ok in url paths and in form query args
-        '~': true,
-        '!': true,
-        '*': true,
-        '(': true,
-        ')': true,
-        '-': true,
-        _: true,
-        '.': true,
-        ',': true,
-        ':': true,
-        '@': true,
-        $: true,
-        "'": true,
-        '/': true,
-    };
-
-    /*
-     * we divide the uri-safe glyphs into three sets
-     *   <rison> - used by rison                         ' ! : ( ) ,
-     *   <reserved> - not common in strings, reserved    * @ $ & ; =
-     *
-     * we define <identifier> as anything that's not forbidden
-     */
-
-    /**
-     * punctuation characters that are legal inside ids.
-     */
-    // this var isn't actually used
-    //rison.idchar_punctuation = "_-./~";
-
-    (function () {
```

Comment:
I was very confused what the point of this function was. `rison.not_idchar` is immediately set to something else afterward so I ended up deleting.